workalendar/europe/netherlands.py

```diff
@@ -15,9 +15,7 @@ class Netherlands(WesternCalendar):
     include_whit_monday = True
     include_boxing_day = True
 
-    FIXED_HOLIDAYS = WesternCalendar.FIXED_HOLIDAYS + (
-        (5, 5, "Liberation Day"),
-    )
+    FIXED_HOLIDAYS = WesternCalendar.FIXED_HOLIDAYS
```

Comment:
probably unnecessary here, since the `Netherlands` class inherits directly from the `WesternCalendar` class.